### DISCUSSION TITLE
Where can I find a ArgoCD Github Action to deploy a service to a Cluster?

### DISCUSSION THREAD

**Discussion by kaihendry** [argoproj/argo-cd#9847] [Q&A] [answered]
Hi, I'm looking for a "Gitops" Github Action / workflow that essentially deploy a service like [fake-service](https://github.com/nicholasjackson/fake-service/issues/75) to a k8s cluster.

![image](https://user-images.githubusercontent.com/765871/176847986-01a511d1-67da-47d3-9acb-4a15601b7af7.png)

My expectation is just to enable `aws eks update-kubeconfig --name mycluster` the workflow and somehow pass the service to ArgoCD or what am I missing?

**Comment by kaihendry** [comment, marked as answer]
Ok, I think this might be a fundamentally a silly question since IIUC ArgoCD is supposed to poll the Git repository. Though I was half expecting a workflow hook at the very least!

**Reply by venkatamutyala** [reply]
You can speed this up with webhooks:

https://argo-cd.readthedocs.io/en/stable/operator-manual/webhook/

**Reply by kaihendry** [reply]
I see, so no need for a workflow at all. Thanks!
